python_modules/dagster/dagster/_core/storage/event_log/schema.py

```diff
@@ -137,6 +137,25 @@
     db.Column("create_timestamp", db.DateTime, server_default=get_current_timestamp()),
 )
 
+AssetChecksTable = db.Table(
+    "asset_checks",
+    SqlEventLogStorageMetadata,
+    db.Column(
+        "id",
+        db.BigInteger().with_variant(sqlite.INTEGER(), "sqlite"),
+        primary_key=True,
+        autoincrement=True,
+    ),
+    db.Column("name", db.Text),
+    db.Column("asset_key", db.Text),
+    db.Column("status", db.Text),
+    db.Column("run_id", db.Text),
+    db.Column("result_body", db.Text),
+    db.Column("assigned_timestamp", db.DateTime),
```

Comment:
I think in the ui we probably want to show the timestamp when the check started evaluating? Vs when it finished, or when the run started

Reply:
Is this the same timestamp that would be assigned to the corresponding event in the event log? If so, a more specific name would be helpful, because this record will exist before that event exists.